nixos/tests/php/default.nix

```diff
@@ -5,4 +5,5 @@
   fpm = import ./fpm.nix { inherit system pkgs; };
   httpd = import ./httpd.nix { inherit system pkgs; };
   pcre = import ./pcre.nix { inherit system pkgs; };
+  unit = import ./unit.nix { inherit system pkgs; };
```

Comment:
Is it actually right to put `unit` in the `php` attribute in the tests?

It may be, I'm not saying it isn't.

I'm just thinking that unit supports other languages as well. Maybe you want tests for those as well?

Reply:
I use only php, for other languages ​​I don’t know how to write tests.

Reply:
Moved test to web-servers/unit-php.nix